ActivitiesAndIntents/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="net.boondockradio.activitiesandintents.MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        android:layout_alignParentTop="true"
+        android:text="@string/app_name"/>
+    
+    <Button
+        android:id="@+id/activity1"
```

Comment:
about naming.
We like to add a prefix to the id.
Button - `btn_`
TextView - `txt_`
*Layout - `layout_`
ImageView - `img_`
etc...

so here `android:id="@+id/btn_activity1"